index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
+var colorConvert = require('color-convert');
+var supportsColor = require('supports-color');
+
+supportsColor = supportsColor || {};
+supportsColor.level = supportsColor.level || 0;
+
+colorConvert.rgb.rgb = function rgb (rgb) {
+	return rgb;
+};
+
+function getConverter (color) {
+	var has = function has (name) {
+		return color.hasOwnProperty(name);
+	}
+	switch (true) {
```

Comment:
I always prefer longer `if`-statements than this kind of tricks, but that's code style, at the end. @sindresorhus ?